tests/Math.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+*/
+
+declare(strict_types=1);
+
+namespace pocketmine\math\tests;
+
+
+use pocketmine\math\Math as MathAbstract;
+
+/**
+ * Class for the Math abstract class
+ */
+class Math extends MathAbstract{
```

Comment:
I'm not sure I see the point of this?

Reply:
Firstly I create the ```pocketmine\math\tests\Math``` class and let it extend the ```pocketmine\math\Math``` class to let ```pocketmine\math\tests\MathTest``` can do unit test for the abstract class ```pocketmine\math\Math```.

Reply:
I don't see why this is necessary - it's not instantiated anyway.

Reply:
To do the unit test for ```pocketmine\math\Math```, I think it should instantiate.
Otherwise it will not do this unit test.

Reply:
The test is not instantiating the `Math` class anywhere - it's only ever used for static function access. Excuse me if I'm missing the point, but this class is not instantiated anywhere I can see in these tests.

Reply:
This ```Math``` class is used in the ```MathTest``` class.

Reply:
... but you still don't _instantiate_ it, you only statically access its methods.

Reply:
Thank you for your explanation and I know that.
I remove those non-instantiate abstract classes.